packages/common/package.json

```diff
@@ -30,12 +30,15 @@
     "browser.js"
   ],
   "dependencies": {
+    "@eth-optimism/hardhat-ovm": "^0.2.2",
     "@ethersproject/bignumber": "^5.0.5",
     "@google-cloud/kms": "^0.3.0",
     "@google-cloud/storage": "^2.4.2",
     "@maticnetwork/maticjs": "^2.0.40",
+    "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers@^0.3.0-beta.10",
```

Comment:
I've had trouble getting `getAbi` or `getArtifact` to work when running scripts on `optimism`:

```
TypeError: artifacts.require is not a function
      at getTruffleContractTest (dist/index.js:108:21)
      at getAbiTest (dist/index.js:144:29)
      at Context.<anonymous> (test/optimism/Finder.js:26:41)
      at processImmediate (internal/timers.js:461:21)
```

Reply:
Perhaps this is because `compile --network optimism` builds contracts in `artifacts-ovm` and caches compile information in `cache-ovm`